### PR TITLE
UI: Updates year to current year in the footer

### PR DESCRIPTION
**Pull request by pratap0007**
# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if functionality changed/added)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/main/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/main/CONTRIBUTING.md) for more details._

### CODE REVIEW

**Comment by piyush-garg** [comment]
Is it fine to do this? cc @vdemeester

File: ui/src/containers/App/__snapshots__/App.test.tsx.snap [hunk 1]
```diff
@@ -307,7 +307,9 @@ exports[`App should render the component correctly and match the snapshot 1`] =
                             <div className=\\"pf-l-grid__item pf-m-12-col hub-footer-description\\">
                               <Text>
                                 <p data-pf-content={true} className=\\"\\">
-                                  © 2020 The Linux Foundation®. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our
+                                  © 
+                                  2021
```
Comment (PuneetPunamiya) [inline ui/src/containers/App/__snapshots__/App.test.tsx.snap:311]: Shouldn't we use `new Date().getFullYear()` here as well

Comment (pratap0007) [reply ui/src/containers/App/__snapshots__/App.test.tsx.snap:311]: I think , we can't do in the snapshot

Comment (PuneetPunamiya) [reply ui/src/containers/App/__snapshots__/App.test.tsx.snap:311]: So the snapshot will change once the year changes right ??

Comment (pratap0007) [reply ui/src/containers/App/__snapshots__/App.test.tsx.snap:311]: yeah

**Pull request merged**